faq/coldstorage.mdx

```diff
@@ -43,10 +43,6 @@ We recommend that you use Glacier to archive data that you may need only once or
 </Message>
 
 Find out more about [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier) our dedicated page.
```

Comment:
```suggestion
Find out more about [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier).
```

Reply:
Or Find out more about [how to restore an object from glacier](/storage/object/how-to/restore-an-object-from-glacier) in our dedicated page.